src/App.tsx

```diff
@@ -15,36 +16,113 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  alphabetically = 'alphabetically',
+  byLength = 'length',
+  default = '',
+}
+
+interface SortParams {
+  sortField: SortType;
+  isReversed: boolean;
+}
+
+function getPrepereGoods(
```

Comment:
There is a typo in the function name `getPrepereGoods`. It should be `getPrepareGoods` to correctly spell 'prepare'.